### DISCUSSION TITLE
Spare PCBs available (PCB_V2B)

### DISCUSSION THREAD

**Discussion by DJSGXII** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#658] [General]
I have some spare parts (not soldered), could sell a kit for 5 eur + shipping cost (Only need to add NodeMCU yourself). Three kits available (in Netherlands), if interested, reply with email address and we can then sort out the details.

**Comment by bailey-tv** [comment]
I would be interested, UK based, james@bailey.tv

**Comment by Ginoooke** [comment]
I am also interested. Do you still have a kit available?

**Reply by DJSGXII** [reply]
Yes I have, just post your email here and I will send you a message. You can remove your email post afterwards ;)

**Comment by mbotty** [comment]
Hi. Do you still have a kit available?

**Reply by DJSGXII** [reply]
Yes I have, just post your email here and I will send you a message. You can remove your email post afterwards ;)

**Comment by DJSGXII** [comment]
ONLY ONE KIT LEFT!!

**Reply by Botap** [reply]
Hi, do you still have this one?
I am based in NL

**Reply by Botap** [reply]
Hoi, je kan mij contacten op

**Reply by pentoo1989** [reply]
is still a kit there?

**Reply by DJSGXII** [reply]
No sorry, my stock has been sold.

**Comment by ashb** [comment]
I have some spare v2 (not v2b) if anyone want's one. I'm in uk, but happy to ship to europe too. Board, headers, ttl convert, 4pin wires.

Email is in my github profile.
